.expeditor/config.yml

```diff
@@ -78,9 +78,9 @@ github:
    - "Expeditor: Bump Major Version"
 
 release_branches:
-  - inspec-6:
-     version_constraint: 6.*
   - main:
+     version_constraint: 6.*
```

Comment:
We want to change the version constrain on main to be a dummy value, `99.*` or similar. We do not want to main to match anything at the moment. Main will be temporarily unreleasable until we merge a PR from the inspec-6 branch making main match 6.*